libs/sentry.py

```diff
@@ -27,9 +27,12 @@ def __init__(self, base_url, auth_token):
         self.__token = auth_token
         self.__session = requests.Session()
 
+    @retry(requests.exceptions.HTTPError, tries=2, delay=1)
```

Comment:
@lululombard Could you make these values (`tries` & `delay`) configurable as proposed on #23 ?

Reply:
Will do!

Reply:
Another thing that would be good is to retry only on `HTTP 429` instead of retry on any HTTP error `5xx` as mentioned on https://github.com/italux/sentry-prometheus-exporter/pull/23#issuecomment-922543930, retrying on any HTTP error can make debugging harder, what you think?

Maybe you should also add a backoff here, something like:
```suggestion
    @retry(requests.exceptions.HTTPError, tries=2, delay=1, backoff=2, jitter=0.5)
```

Reply:
The backoff and jitter is a good idea, for the handling of exceptions, I think it's fine to retry on anything as exceptions are logged with the error code [cf screenshot](https://user-images.githubusercontent.com/2182934/132733103-b0232917-a5ee-443d-adc4-7755f474534c.png)